extensions/rich_text_components/Svgdiagram/protractor.js

```diff
@@ -77,11 +77,17 @@ var customizeComponent = async function(modal, shape) {
   await waitFor.visibilityOf(
     modal.element(by.css('.protractor-test-saved-diagram-container')),
     'Diagram container not visible');
+  var altTextInputElement = (
+    element(by.css('[placeholder = "Description of the diagram"]')));
+  await altTextInputElement.sendKeys(altText);
```

Comment:
Please check that this has loaded first. You can use our new actions utilities to do this easily:

https://github.com/oppia/oppia/blob/1578d4b16174127fda917983c707ba6d9d93ce1c/core/tests/protractor_utils/action.js#L45